src/main/drivers/pitotmeter/pitotmeter_adc.c

```diff
@@ -67,6 +67,7 @@ static void adcPitotCalculate(pitotDev_t *pitot, float *pressure, float *tempera
 bool adcPitotDetect(pitotDev_t *pitot)
 {
     pitot->delay = 10000;
+    pitot->calibThreshold = 0.00001f;   // TODO :: should be tested !!!
```

Comment:
This constant seems to be repeated all over the place.

It probably would be nice to add a #define for that.

Reply:
Not true. See ``pitotmeter_ms4525.c`` and line 147:
```
    pitot->calibThreshold = 0.00005f;   // noisy sensor
```
BTW, the same can be said about line 69: ``pitot->delay = 10000;``